e2etest/factory.go

```diff
@@ -60,8 +63,19 @@ func (TestResourceFactory) GetFileServiceURL(accountType AccountType) azfile.Ser
 	if err != nil {
 		panic(err)
 	}
-	pipeline := azfile.NewPipeline(credential, azfile.PipelineOptions{})
-	return azfile.NewServiceURL(*u, pipeline)
+	// Closest to API goes first; closest to the wire goes last
+	f := []pipeline.Factory{
```

Comment:
Can we not re-use the integrated new pipeline functions in AzCopy? Is there a good reason to duplicate this?

Reply:
I can reuse the pipeline creation logic in ste